sdk/resourcemanager/azure-resourcemanager/src/samples/java/com/azure/resourcemanager/containerservice/generated/ManagedClustersDeleteSamples.java

```diff
@@ -23,6 +23,6 @@ public static void deleteManagedCluster(com.azure.resourcemanager.AzureResourceM
             .manager()
             .serviceClient()
             .getManagedClusters()
-            .delete("rg1", "clustername1", null, com.azure.core.util.Context.NONE);
+            .delete("rg1", "clustername1");
```

Comment:
Wonder why the change here.. `azure-resourcemanager` still depends on the stable version of containerservice right?

Reply:
Yes.

This was the "break" I've mentioned. The preview api-version has 1 more optional parameter in this API (therefore BuildFromSource fails -- this depends on code, not the stable).

Reply:
I see.. `BuildFromSource` will build dependent libraries from source even when it's stable version.